nixos/modules/services/databases/cockroachdb.nix

```diff
@@ -70,10 +70,12 @@ in
           like datacenter.  The tiers and order must be the same on all nodes.
           Including more tiers is better than including fewer. For example:
 
+          <literal>
               country=us,region=us-west,datacenter=us-west-1b,rack=12
               country=ca,region=ca-east,datacenter=ca-east-2,rack=4
 
               planet=earth,province=manitoba,colo=secondary,power=3
+          </literal>
```

Comment:
Wouldn't that be one single line? Like eg. if used for the example attribute of `mkOption`:

```nix
mkOption {
  type = types.nullOr types.str;
  default = null;
  example = "country=us,region=us-west,datacenter=us-west-1b,rack=12,"
          + "country=ca,region=ca-east,datacenter=ca-east-2,rack=4,"
          + "planet=earth,province=manitoba,colo=secondary,power=3";
  description = ''
    ...
  '';
}
```

Reply:
Ah, maybe. I'll double check this too. `<example>` doesn't seem to work though (some kind of nested `<para>` conflict, I think?) but I'll look around for this, too.

Reply:
[`<example/>`](https://tdg.docbook.org/tdg/5.0/example.html) work within `<para/>`, but you'll need a `<title/>` or `<info/>` child. [`<informalexample/>`](https://tdg.docbook.org/tdg/5.0/informalexample.html) might be better suited here. Or as said: It's maybe even better to use the `example` attribute of `mkOption`.